docker/generic/start_proxy.py

```diff
@@ -155,24 +138,28 @@ def make_argparser():
            --service, --version, and --rollout_strategy.
         ''')
 
+    # TODO(b/149050012): Update this help text when `--backend_http_protocol`
+    #  is introduced.
     parser.add_argument(
         '-a',
         '--backend',
         default=DEFAULT_BACKEND,
-        help=''' Change the application server address to which ESPv2
-        proxies requests. Default value: {backend}. For HTTPS backends,
-        please use "https://" prefix, e.g. https://127.0.0.1:8082.
-        For HTTP/1.x backends, prefix "http://" is optional.
-        For GRPC backends, please use "grpc://" prefix,
-        e.g. grpc://127.0.0.1:8082.'''.format(backend=DEFAULT_BACKEND))
-
-    parser.add_argument(
-        '--backend_protocol',
-        default=None,
-        help='''Backend Protocol. Overrides the protocol in --backend.
-        Choices: [http|https|grpc|grpcs].
-        Default value: http.''',
-        choices=['http', 'https', 'grpc', 'grpcs'])
+        help='''
+        Specify the backend application server address.
+        Default value: {backend}.
```

Comment:
->  
Specify the backend application server address, when using ESPv2 as sidecar.
Default value: {backend}.  When override, following the same format,  with valid schemes `http, https, grpc, grpcs`.

Reply:
Done